atomicfu-gradle-plugin/build.gradle

```diff
@@ -24,76 +24,12 @@ dependencies {
     // Atomicfu compiler plugin dependency will be loaded to kotlinCompilerPluginClasspath
     // Atomicfu plugin will only be applied if the flag is set kotlinx.atomicfu.enableJsIrTransformation=true
     compileOnly "org.jetbrains.kotlin:atomicfu:$kotlin_version"
+    runtimeOnly "org.jetbrains.kotlin:atomicfu:1.7.0"
```

Comment:
Please add the comment describing why we have two different dependencies on two different version of the same artifact.
It would be also nice to describe why you chose 1.7.0 instead of, lets say, 1.6.20.

Reply:
Done)

Reply:
Thanks!